src/views/Home.jsx

```diff
@@ -66,9 +55,8 @@ export function Home({ data, setListPath, userId, userEmail }) {
 						name={item.name}
 						email={userEmail}
 						path={item.path}
-						setListPath={handleListClick}
-						selected={selectedList === item.path} // Checks if the list is selected
-						className="SingleList"
+						setListPath={setListPath}
+						listPath={listPath} // Pass listPath to SingleList
 					/>
```

Comment:
no need to pass the listPath further, just use it here.
selected={listpath === item.path}

Reply:
I dont really understand...

Reply:
before you were using another duplicated state `selectedList` to check if the list is selected like this :
`selected={selectedList === item.path}`. 

Now you will use the `listPath`, that's it:
`selected={listpath === item.path}`